practice/flutter/flutter/logic.py

```diff
@@ -4,24 +4,42 @@
 
 
 def get_flutts():
-    """Return 10 Flutts for display."""
+    """Return 10 Flutts for display, ordered by timestamp, newest on top.
+
+    >>> models.Flutt(text='Wo', timestamp='2016-09-05T11:00').save()
+    >>> models.Flutt(text='He', timestamp='2016-09-05T10:00').save()
+    >>> get_flutts()  # doctest: +NORMALIZE_WHITESPACE
+    [Flutt(text:'Wo', date:datetime.datetime(2016, 9, 5, 18, 0, tzinfo=<UTC>)),
+    Flutt(text:'He', date:datetime.datetime(2016, 9, 5, 17, 0, tzinfo=<UTC>))]
+    """
     flutt_list = models.Flutt.objects.all().order_by('-timestamp')
     last_item = min(len(flutt_list), 10)
     return flutt_list[:last_item]
 
 
 def get_queried_flutts(string):
-    """Return 10 Flutts filtered flutts for display."""
+    """Return 10 Flutts filtered flutts for display.
+
+    >>> models.Flutt(text='Wo', timestamp='2016-09-05T11:00').save()
+    >>> models.Flutt(text='He', timestamp='2016-09-05T10:00').save()
+    >>> get_queried_flutts('o')  # doctest: +NORMALIZE_WHITESPACE
+    [Flutt(text:'Wo', date:datetime.datetime(2016, 9, 5, 18, 0, tzinfo=<UTC>))]
+    """
     flutt_list = models.Flutt.objects.all().order_by('-timestamp')
     filtered_flutts = flutt_list.filter(text__contains=string)
     if len(filtered_flutts) == 0:
         return None
```

Comment:
Avoid returning None when the empty list will do. The empty list pretty cleanly represents that there are no results and also doesn't require special casing every call to this function to check if you got None.